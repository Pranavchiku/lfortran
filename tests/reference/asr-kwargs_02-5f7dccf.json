{
    "basename": "asr-kwargs_02-5f7dccf",
    "cmd": "lfortran --indent --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/kwargs_02.f90",
    "infile_hash": "8d5322cf8bcf42b96ba5ad60e9410309159c7c0e999797cef4da7d6e",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-kwargs_02-5f7dccf.stdout",
    "stdout_hash": "718c52553f4ffccdac287c82823b43ea5e73a20c4ff0aa985f724a31",
    "stderr": "asr-kwargs_02-5f7dccf.stderr",
    "stderr_hash": "40d20f33525bf836a22a28a733677dab2ab6e2ea658c5ade7c979829",
    "returncode": 0
}