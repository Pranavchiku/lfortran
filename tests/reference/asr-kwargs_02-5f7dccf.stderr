warning: Implied save feature
  --> tests/../integration_tests/kwargs_02.f90:16:9
   |
16 |         class(logger_type), intent(inout) :: self
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/kwargs_02.f90:17:9
   |
17 |         character(*), intent(in) :: filename
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/kwargs_02.f90:18:9
   |
18 |         integer, intent(out), optional :: unit
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/kwargs_02.f90:23:9
   |
23 |         class(logger_type), intent(inout)  :: self
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/kwargs_02.f90:24:9
   |
24 |         integer, intent(in) :: code
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/kwargs_02.f90:25:9
   |
25 |         character(*), intent(in), optional :: filename
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
  --> tests/../integration_tests/kwargs_02.f90:26:9
   |
26 |         integer, intent(out), optional :: unit
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables
